src/day8/re_frame/trace.cljs

```diff
@@ -144,11 +144,19 @@
     (fn [trace]
       (< (:query query) (:duration trace)))))
 
-(defn render-traces [showing-traces]
+(defn render-traces [showing-traces filter-items]
   (doall
     (for [{:keys [op-type id operation tags duration] :as trace} showing-traces]
-      (let [padding   {:padding "0px 5px 0px 5px"}
-            row-style (merge padding {:border-top (case op-type :event "1px solid lightgrey" nil)})
+      (let [hover-trace      (r/atom false)
+            padding          {:padding "0px 5px 0px 5px"}
+            row-style        (merge padding {:border-top (case op-type :event "1px solid lightgrey" nil)})
+            op-name          (if (= PersistentVector (type (js->clj operation)))
```

Comment:
We may be better off making the op-name always a vector with one or more params, rather than a keyword or a vector.